components/movementsensor/merged/merged.go

```diff
@@ -101,14 +101,15 @@ func (m *merged) Reconfigure(ctx context.Context, deps resource.Dependencies, co
 
 		for _, name := range names {
 			ms, err := movementsensor.FromDependencies(deps, name)
+			msName := ms.Name().ShortName()
 			if err != nil {
-				logger.Debugf("error getting sensor %v from dependencies", ms.Name().ShortName())
+				logger.Debugf("error getting sensor %v from dependencies", msName)
 				continue
 			}
 
 			props, err := ms.Properties(ctx, nil)
 			if err != nil {
-				logger.Debugf("error in getting sensor %v properties", ms.Name().ShortName())
+				logger.Debugf("error in getting sensor %v properties", msName)
```

Comment:
Discussion question: we can use `Debugf` like this to format things so they're easy for a human to read, or we can use `Debugw` to format it so it's easy for a computer (Google Computing Services?) to read (e.g., `logger.Debugw("error getting sensor properties", "sensor_name", msName)`). Which way is preferred?

Reply:
human-readable for this one since it's a reconfiguration log I think